### DISCUSSION TITLE
Track custom properties with Plausible

### DISCUSSION THREAD

**Discussion by luminous8** [nuxt/scripts#282] [General]
Hi,

According to Plausible doc, custom props are supposed to be in the script tag
`<script defer event-author="John Doe" data-domain="yourdomain.com" src="https://plausible.io/js/script.pageview-props.js"></script>`
https://plausible.io/docs/custom-props/for-pageviews

How is that supposed to work with Nuxt Script?

Thanks!

**Comment by harlan-zw** [comment]
The Plausible registry now supports this directly via `customProperties` (October 2025 format):

```ts
useScriptPlausibleAnalytics({
  scriptId: 'YOUR_SCRIPT_ID',
  customProperties: { author: 'John Doe' },
})
```

For per-event props, use the runtime API: `plausible('Custom Event', { props: { author: 'John Doe' } })`. Closing 🙏
